src/accidental.ts

```diff
@@ -528,17 +565,17 @@ export class Accidental extends Modifier {
     L('Rendering: ', type, accX, accY);
 
     if (!cautionary) {
-      glyph.render(context, accX, accY);
+      glyph.render(ctx, accX, accY);
     } else {
       // Render the accidental in parentheses.
-      parenRight.render(context, accX, accY);
-      accX -= getGlyphWidth(parenRight);
+      parenRight?.render(ctx, accX, accY);
```

Comment:
These are silent failures here. Can you explicitly test them please? (Maybe with `check`?)

(The ones below too.)

Reply:
Done